app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
```

Comment:
The function should first check if the command starts with 'cp' and contains exactly three elements. This is important to ensure the command is valid according to the task requirements.